lib/interactive_viewer.ml

```diff
@@ -104,6 +104,64 @@ let change_summary z_patches : ui Lwd.t =
   in
   W.string ~attr:Notty.A.(fg lightcyan) operation_count
 
+
+(** Side by side diff view implementation **)
+
+type view_mode = SideBySide | Normal
+
+let view_mode = Lwd.var SideBySide
```

Comment:
I think the default should be `Normal` for now.

Reply:
Resolved.